tests/Monolog/Formatter/GelfMessageFormatterTest.php

```diff
@@ -82,10 +83,11 @@ public function testFormatWithFileAndLine()
 
     /**
      * @covers Monolog\Formatter\GelfMessageFormatter::format
-     * @expectedException InvalidArgumentException
      */
     public function testFormatInvalidFails()
     {
+        $this->expectException(\InvalidArgumentException::class);
```

Comment:
The reason the annotations are deprecated is in order to explicitly be added before the class that will throw the exception.. So this must be moved below.. and excactly above the function that will throw the exception

Reply:
Same thing for other places as well.

Reply:
@gmponos thanks for the review. I will update PR accordingly.